src/app/users/views/confirmNewUser.ejs

```diff
@@ -9,7 +9,7 @@
         <h1 class="heading-xlarge">
             Review invite details
         </h1>
-        <p>Please review these details before sending out an invite.  The user will have to confirm their email address and create a password before their account is active.</p>
+        <p>Please review these details before sending an invite.  The user will need to confirm their email address and create a password before their DfE Sign-in becomes active.</p>
```

Comment:
I am not sure 

> before their DfE Sign-in becomes active

Makes sense, as the 'service' is also DfE Sign-in. Perhaps DfE Sign-in account or profile or something?